modules/ROOT/pages/login-platform-from-studio.adoc

```diff
@@ -0,0 +1,18 @@
+= Login to Anypoint Platform from Studio
```

Comment:
```suggestion
= Log in to Anypoint Platform from Studio
```
Login = noun
Log in = verb

Reply:
Hey - sorry to dissent, but I just reviewed some changes Valk made to the IAM docs and it stuck in my head... For Anypoint Platform, we use "sign in" (same noun/verb distinction ask 25 says) because that's what the UI says. I'm not sure what the Studio option is, though.
https://docs.mulesoft.com/access-management/managing-your-account#sign-in-to-anypoint-platform

Reply:
Yes. Ultimately, you are signing in to Anypoint Platform. Studio needs to fix its UI. 
![image](https://github.com/mulesoft/docs-studio/assets/36547707/7167b594-ae94-4b3a-9d62-629c60a42d40)

Reply:
Thanks for your collaboration! 
I contacted the Studio team to fix the UI.